src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/ScegliSpidPGPage.java

```diff
@@ -37,6 +37,7 @@ public void waitLoadScegliSpidPGPage() {
     }
 
     public void clickTestButton() {
+        getWebDriverWait(60).withMessage("Il bottone TEST non è cliccabile nella login").until(ExpectedConditions.elementToBeClickable(this.testButton));
```

Comment:
inserire messaggio

Reply:
esiste già il messaggio sia prima di chiamare il metodo che in caso va n errore